src/components/canvas-tools/templates/Circle.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div>
+    <p>Outline colour</p>
+    <v-color-picker
+      v-model="circleColour"
+      :swatches="swatches"
+      mode="hexa"
+      show-swatches
+      hide-canvas
+      flat
+      hide-inputs
+      hide-mode-switch
+    />
+    <v-slider
+      prepend-icon="fa-ruler-vertical"
+      :hint="circleSizeHint"
+      v-model="circleSize"
+      :max="6"
+      :step="1"
+      ticks="always"
+    />
+    <p>Fill colour</p>
+    <v-color-picker
```

Comment:
Can we pull this colour picker out and import it into all the tools. I don't think any other tools will have their own colours, just what ever we set?